test/unit/pipeline/test_train_model_sklearn.py

```diff
@@ -56,6 +61,25 @@ def test_split_data_train_validation_test_period(self):
         self.assertEqual(len(validation_data), 1296)
         self.assertEqual(len(test_data), 1)
 
+    def test_train_pipeline_happy(self):
+        """Test if happy flow of the train pipeline
+
+        The input data should not contain features (e.g. T-7d),
+        but it can/should include predictors (e.g. weather data)"""
+
+        # Run the this should return a model and report
+        with self.assertLogs() as captured:
+            model, report = train_model_pipeline(
+                pj=self.pj,
+                input_data=self.train_input,
+                # check_old_model_age=False,
+                # compare_to_old=False,
+            )
+        # Check if a new model was stored based on logging
+        assert captured.records[-1].getMessage() == "Fitted a new model, not yet stored"
+        assert str(type(model)) == "<class 'xgboost.sklearn.XGBRegressor'>"
```

Comment:
This seems very brittle. It would be better to compare the type (not the string) to some kind of base estimator I think.

Reply:
Thanks, it was changed to your suggestion